app/services/actual/import.rb

```diff
@@ -35,6 +36,8 @@ def import_row(row, index)
 
       importer.errors.each do |attr_name, (value, message)|
         add_error(index, attr_name, value, message)
+
+        @invalid_with_comment = true if row["Comment"].present?
```

Comment:
Can this be simplified by removing the `@invalid_with_comment = false` and changing this to `@invalid_with_comment = row["Comment"].present?`

Reply:
Or will this throw an error if the instance variable is absent and the attr_accessor is called 🤔

Reply:
So the idea here is that we set it to `false` when the importer is instantiated (so it would never be absent when we get here), then the first row that has a comment sets it to `true` and then it never gets set to `false`. If we set it on every row, we'll forget that an earlier row had a comment in. But I think we can actually do `@invalid_with_comment ||= row["Comment"].present?`, which simplified the syntax but doesn't overwrite an earlier `true` row with a later `false`